scripts/run_adv_im.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+# To avoid complexisity this script will only run one Adv_IM model at a time
+if [[ $# -lt 2 ]];then
+    # more than one args are provided, exit and warn user
+    #echo "this script only runs on one Adv_IM model, please make sure only one is provided"
+    echo "please provide 1. model name 2. list of events"
+    exit 1
+fi
+ADV_IM_NAME=$1
+LIST_EVENTS_F=$2
+
+# some constant variables, only update these when version changes
+VALIDATION_VERSION=v20p5p8
+BBbin_root_dir=/nesi/nobackup/nesi00213/RunFolder/ykh22/Adv_IM/$VALIDATION_VERSION
+SIM_DATA_DIR=/nesi/nobackup/nesi00213/RunFolder/ykh22/Adv_IM/$VALIDATION_VERSION/Data/
+TASK_CONFIG_TEMPLATE=$BBbin_root_dir/task_config.yaml
+OBS_DATA_DIR=/nesi/nobackup/nesi00213/RunFolder/ykh22/Adv_IM/ObservedGroundMotions
```

Comment:
These should be moved to /nesi/project, rather than left in nobackup?

Reply:
i'll run a du to check the total size on BBs and Models. If its too large we might have to keep it there.

Reply:
https://github.com/ucgmsim/slurm_gm_workflow/pull/337/commits/d1f96e122d5eb707a01e9182039847c971c9d9a7